torch/utils/mkldnn.py

```diff
@@ -104,6 +104,56 @@ def __setstate__(self, state):
         self.bias = state[1].to_mkldnn()
         self.training = state[2]
 
+class MkldnnConv3d(torch.jit.ScriptModule):
```

Comment:
FYI I believe this is kind of a legacy API as we now compile nn Modules recursively. Cc @eellison

Reply:
yea, correct, better to inherit from `torch.nn.Module`, you shouldn't need any other changes

Reply:
Thanks, I will change it,  and also for other case at next step.

Reply:
@eellison , if it is inherited from **torch.nn.Module**,  there will has a problem for **torch.jit.save** method, because for a MKLDNN module, the parameters are MKLDNN tensor which are opaque tensors(do not have storage), we will first call **.to_dense()** at  **__getstate__**  to save this script module.  I will changed until this problem can be sovled. thanks!